pyproject.toml

```diff
@@ -1,3 +1,14 @@
 [tool.black]
 line-length = 100
 target-version = ['py37']
+
+[tool.isort]
+skip = "monkey/monkey_island/cc/ui"
+known_first_party = "common,infection_monkey,monkey_island"
+line_length = 100
+### for compatibility with black
+multi_line_output = 3
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+ensure_newline_before_comments = true
```

Comment:
Based on [this](https://pycqa.github.io/isort/docs/configuration/black_compatibility/), wouldn't this be enough?
```
[tool.isort]
profile = "black"
multi_line_output = 3
```

Reply:
It won't pick up our black config. It'll use a predefined one (which doesn't match ours). See [this](https://pycqa.github.io/isort/docs/configuration/profiles/).